api/src/main/java/org/openmrs/obs/handler/AbstractHandler.java

```diff
@@ -25,6 +25,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import liquibase.util.file.FilenameUtils;
```

Comment:
This is the wrong `FilenameUtils`, you should use the Apache one.

Reply:
I could have sworn I used Apache. I suppose I selected the wrong one accidentally in Eclipse when it asked me and did not notice.